examples/auth_example/auth_example_client/lib/src/protocol/client.dart

```diff
@@ -30,10 +34,16 @@ class EndpointExample extends _i1.EndpointRef {
 
 class Modules {
   Modules(Client client) {
-    auth = _i3.Caller(client);
+    serverpod_auth_email = _i3.Caller(client);
+    serverpod_auth_email_endpoint = _i4.Caller(client);
+    auth = _i5.Caller(client);
   }
 
-  late final _i3.Caller auth;
+  late final _i3.Caller serverpod_auth_email;
+
+  late final _i4.Caller serverpod_auth_email_endpoint;
```

Comment:
This would of course be a breaking change for the client.
A small fix for sure (if we were to get the endpoint mapping & model parsing to get rewired on the backend to support older clients), but nonetheless it'd require a few 1-line changes if the app (client) uses its own UI instead of the `serverpod_email_flutter` package with pre-built component (in which case one would still required to update those packages in unison though).